shop/src/pages/admin/settings/shipping/_Edit.js

```diff
@@ -63,7 +63,10 @@ const EditShippingMethod = ({ onClose, shippingZone }) => {
     ...initialState,
 
     ...shippingZone,
-    amount: shippingZone ? (shippingZone.amount / 100).toFixed() : 0,
+    amount:
```

Comment:
@nick There is a merge conflict on this line. Just noticed you changed this line to `amount` to fix the same error on master. We actually store the shipping price in cents, so setting this to just `amount` would break editing a shipping option. Or are we going to start storing in $ instead of cents?

Reply:
Nvm, didn't notice the lodash get on the top